packages/react/cypress/component/auto/form/PolarisFileInput.cy.tsx

```diff
@@ -0,0 +1,348 @@
+import React from "react";
+import { getStadiumModelMetadata, getStadiumRecord } from "../../../../spec/auto/support/stadiumModel.js";
+import { PolarisAutoFileInput } from "../../../../src/auto/polaris/PolarisAutoFileInput.js";
+import { PolarisAutoForm } from "../../../../src/auto/polaris/PolarisAutoForm.js";
+import { api } from "../../../support/api.js";
+import { PolarisWrapper } from "../../../support/auto.js";
+
+describe("PolarisFileInput", () => {
+  const getSubmitButton = () => {
+    return cy.get("form [type=submit][aria-hidden!=true]");
+  };
+
+  const interceptModelCreateActionMetadata = (validations: any[]) => {
```

Comment:
Can this be moved to a new file or does it not owrk when it is not done in the test file?

Reply:
No, but let me fix forward and have a separate PR to improve these Cypress util functions